src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1244,6 +1244,19 @@
     /**
      * The {@code ]} symbol.
      *
+     * <p>For example:</p>
+     * <pre>
+     * int a[];
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--TYPE -&gt; TYPE
```

Comment:
hell, it seems like it should start with `VARIABLE_DEF`, please update

Reply:
@timurt done

Reply:
sorry, i just noticed that I wrote 'hell' instead of 'hello' :|

Reply:
No no its okay...XD